docs/guides/beginnerCLI.rst

```diff
@@ -0,0 +1,276 @@
+    The Magnetic Fields - Distortion - Old Fools
+    beet ls hissing gronlandic
+    of Montreal - Hissing Fauna, Are You the Destroyer? - Gronlandic Edit
+    beet ls bird
+    The Knife - The Knife - Bird
+    The Mae Shi - Terrorbird - Revelation Six
+    beet ls album:bird
+    The Mae Shi - Terrorbird - Revelation Six
+
+By default, a search term will match any of a handful of :ref:`common
+attributes <keywordquery>` of songs.
+(They're
+also implicitly joined by ANDs: a track must match *all* criteria in order to
+match the query.) To narrow a search term to a particular metadata field, just
+put the field before the term, separated by a : character. So ``album:bird``
+only looks for ``bird`` in the "album" field of your songs. (Need to know more?
+:doc:`/reference/query/` will answer all your questions.)
+
+The ``beet list`` command also has an ``-a`` option, which searches for albums instead of songs::
+
+    beet ls -a forever
+    Bon Iver - For Emma, Forever Ago
+    Freezepop - Freezepop Forever
+
+There's also an ``-f`` option (for *format*) that lets you specify what gets displayed in the results of a search::
+
+    beet ls -a forever -f "[format] album (year) - artist - title"
+    [MP3] For Emma, Forever Ago (2009) - Bon Iver - Flume
+    [AAC] Freezepop Forever (2011) - Freezepop - Harebrained Scheme
+
+In the format option, field references like `$format` and `$year` are filled
+in with data from each result. You can see a full list of available fields by
+running ``beet fields``.
+
+Beets also has a ``stats`` command, just in case you want to see how much music
+you have::
+
+    beet stats
+    Tracks: 13019
+    Total time: 4.9 weeks
+    Total size: 71.1 GB
+    Artists: 548
+    Albums: 1094
+
+An example of some of these commands will look like this:
+
+    .. image:: ../_static/Images/extraCommands.png
+        :width: 600
+
+If you need more of a walkthrough on configuring and importing libraries, you can read a more in depth and illustrated one `on the
+beets blog <https://beets.io/blog/walkthrough.html>`_.
+
+Keep Playing
+------------
+
+This is only the beginning of your long and prosperous journey with beets. To
+keep learning, take a look at :doc:`advanced` for a sampling of what else
+is possible. You'll also want to glance over the :doc:`/reference/cli` page
+for a more detailed description of all of beets' functionality.  (Like
+deleting music! That's important.)
+
+Also, check out :doc:`beets' plugins </plugins/index>`.  The
+real power of beets is in its extensibility---with plugins, beets can do almost
+anything for your music collection.
+
+You can always get help using the ``beet help`` command. The plain ``beet help``
+command lists all the available commands; then, for example, ``beet help import`` gives more specific help about the ``import`` command.
+
+Want to stay updated on beets? Follow `@b33ts`_ on Twitter to hear about progress on
+new versions.
+
+.. _@b33ts: https://twitter.com/b33ts
```

Comment:
I highly suggest using the Fediverse (Mastodon) link instead of Twitter as the primary way to hear about news about Beets. Reasons:

1. Fediverse/Mastodon is built on open source, just like Beets.
2. One of the more unsavory moves at Twitter is that they now require everyone to log in just to _**see**_ Beets-related news/tweets, whereas anyone can read Beets-related news on the [Fediverse](https://fosstodon.org/@beets) without having to log in.

If it were up to me, I would remove the Twitter link entirely from this documentation, just based on (2) above. But if you are determined to include a Twitter link, at the very least I recommend swapping the two, so that the Fediverse link is primary and the part below says:

```
Please let us know what you think of beets via `the discussion board`_ or `Twitter`_.
```

Reply:
Yeah, phase out Twitter. Mastodon is at least semi-active now and you'll get version release toots whenever we do a release now!